packages/gatsby-theme-ui-blog/tsconfig.json

```
@@ -0,0 +1,3 @@
+{
+  "extends": "../core/tsconfig"
+}
```

This lands in user's node_modules. I think we can't reference core's tsconfig there.

```suggestion
{
  "compilerOptions": {
    "resolveJsonModule": true,
    "esModuleInterop": true,
    "moduleResolution": "node",
    "strict": true,
    "jsx": "preserve"
  }
}
```